study.md

```diff
@@ -66,7 +68,7 @@ Which ActiveRecord method finds all of the records of a certain type (or
 entity)?
 
 ```md
-<!-- your response here -->
+where
```

Comment:
`#where` will work with no filtering arguments.  `#all` is the one that will retrieve all records of a particular collection.